pkgs/os-specific/darwin/swift-corelibs/pure-corefoundation-hook.sh

```diff
@@ -0,0 +1,7 @@
+usePureCoreFoundation() {
+# Avoid overriding value set by the impure CF
+    if [ -z "${NIX_COREFOUNDATION_RPATH:-}" ]; then
+        export NIX_COREFOUNDATION_RPATH=@out@/Library/Frameworks
```

Comment:
I'm not very familiar with setup hooks. `@out@` is substituted right before running them?

Reply:
It's substituted at build time.